src/test/java/com/aggregated/StringUtilsTest.java

```diff
@@ -0,0 +1,39 @@
+package com.aggregated;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class StringUtilsTest {
+
+    @Test
+    public void testIsNonBlankWithBlankString() {
+        assertFalse(StringUtils.isNoneBlank(""));
+    }
+
+    @Test
+    public void testIsNonBlankWittNullString(){
```

Comment:
Please consider templating the syntax room,
be it
```
(){
```
or
```
() {
```

Reply:
My bad, I'm forgot to apply code format for it. Done for now.

Reply:
Thanks @hienkwan !